beangulp/importers/csvbase.py

```diff
@@ -168,17 +169,28 @@ def __new__(mcs, name, bases, dct):
         return super().__new__(mcs, name, bases, others)
 
 
+class EntryOrder(Enum):
+    Infer = 0
+    ExecutionOrder = 1
+    """Entries are listed from first to last"""
+    ReverseExecutionOrder = 2
+    """Entries are listed from last to first"""
```

Comment:
```suggestion
```suggestion
class Order(Enum):
    ASCENDING = 1
    """Entries are listed in chronological order."""
    DESCENDING = 2
    """Entries are listed in reverse chronological order."""
```
```